bosh-stemcell/lib/bosh/stemcell/stage_collection.rb

```
@@ -47,46 +47,46 @@ def agent_stages
+      when Infrastructure::Alicloud
+        alicloud_stages
+      when Infrastructure::CloudStack
+        cloudstack_stages
+      when Infrastructure::Google
+        google_stages
+      when Infrastructure::OpenStack
+        openstack_stages
+      when Infrastructure::Vsphere
+        vsphere_vcloud_stages
+      when Infrastructure::Vcloud
+        vsphere_vcloud_stages
```

RuboCop (1.86.0)

[warning] Duplicate branch body detected.

(Lint/DuplicateBranch)